lib/web/plug/session.ex

```diff
@@ -21,8 +21,9 @@ defmodule Antikythera.Plug.Session do
   alias Antikythera.Conn
   alias Antikythera.Session
 
-  defun load(conn :: v[Conn.t()], opts :: Keyword.t(String.t() | atom)) :: Conn.t() do
-    key = opts[:key]
+  defun load(conn :: v[Conn.t()], opts :: Keyword.t(String.t() | atom | map)) :: Conn.t() do
+    key             = opts[:key]
+    set_cookie_opts = opts[:set_cookie]
```

Comment:
The default value should also be a valid `Http.SetCookie.options_t`.

```suggestion
    set_cookie_opts = Keyword.get(opts, :set_cookie, %{})
```

Reply:
Also addressed this in e534b4c